server/channels/db/migrations/postgres/000114_user_reporting_changes.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE users ADD COLUMN IF NOT EXISTS lastlogin bigint NOT NULL DEFAULT 0;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS poststats AS
```

Comment:
May this create any "delay" when upgrading the instance?

Reply:
Has this been gone through with the server team then? If so, LGTM.

Reply:
Yes I believe so. I'll leave this conversation open so they can chime in if need be.

Reply:
To reiterate, the time it takes to upgrade doesn't matter. It can be anything. What matters is that there cannot be any query which locks the table, even if it locks the table for a millisecond.